index.pug

```diff
@@ -1,3 +1,14 @@
+title: Create Custom Branded UPI/Bharat QR code with Razorpay MultiQR
+description: Unlimited UPI/Bharat QR codes for merchants. Put your Business Name, Logo on QR codes. Accept digital payments from customers using UPI apps, wallets, cards directly.
+---
+extends /templates/base
+append vars
+  - pageid = 'education'
+  - _class = 'signupcard'
+  - hasTopbar = false
+  - signup_link = 'https://dashboard.razorpay.com/signup'
+  append head
+    link(rel='stylesheet' href='/styles/qr-code.css')
 // Institutions Section Start
 section#institution.container-fluid.sec4
```

Comment:
Please correct the intentation, this will lead to break the page

Reply:
correct this